runtime/lua/man.lua

```diff
@@ -148,8 +148,9 @@ local function highlight_line(line, linenr)
 end
 
 local function highlight_man_page()
-  local mod = vim.api.nvim_eval("&modifiable")
-  vim.api.nvim_command("set modifiable")
+  local buf = vim.api.nvim_get_current_buf()
```

Comment:
Don’t need this, zero stands for the current buffer (and is used just below, in `nvim_buf_get_lines`). (It would actually be cleaner to define something like `vim.NVIM_CURRENT_BUFFER` constant, but there is nothing like this currently.)